### PR TITLE
JBTM-3655 A few changes required for Quarkus LRA TCK runs

### PR DESCRIPTION
**Pull request by xstefank**
https://issues.redhat.com/browse/JBTM-3655

!CORE !TOMCAT !AS_TESTS !RTS !JACOCO !XTS !QA_JTA !QA_JTS_OPENJDKORB !PERF LRA !DB_TESTS !mysql !db2 !postgres !oracle

### CODE REVIEW

**Comment by xstefank** [comment]
Can you please also do a release once this is merged? It would be very useful to have TCK running in Quarkus.

**Review by mmusgrov — APPROVED** [review]
Looks nice, thanks very much.

**Comment by mmusgrov** [comment]
> Can you please also do a release once this is merged? It would be very useful to have TCK running in Quarkus.

Will do, but note that a release is blocked by [JBTM-3656](https://issues.redhat.com/browse/JBTM-3656) at the moment although I do have a PR that will hopefully resolve it.

**Pull request merged**
